### PR TITLE
Amundsen Connector Fix 

### PR DESCRIPTION
**Pull request by ulixius9**
### Describe your changes :

Amundsen Connector Fix for 0.10.1

#
### Type of change :

- [x] Bug fix

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

@open-metadata/ingestion

### CODE REVIEW

File: ingestion/src/metadata/ingestion/source/amundsen.py [hunk 1]
```diff
@@ -105,6 +116,17 @@ def __init__(self, config: WorkflowSource, metadata_config: OpenMetadataConnecti
...
+            "glue": DatabaseServiceType.Glue.value,
+            "snowflake": DatabaseServiceType.Snowflake.value,
+            "athena": DatabaseServiceType.Athena.value,
+            "bigquery": DatabaseServiceType.BigQuery.value,
+            "db2": DatabaseServiceType.Db2.value,
+            "druid": DatabaseServiceType.Druid.value,
+            "delta": DatabaseServiceType.DeltaLake.value,
+            "salesforce": DatabaseServiceType.Salesforce.value,
+            "oracle": DatabaseServiceType.Oracle.value,
+        }
```
Comment (pmbrull) [inline ingestion/src/metadata/ingestion/source/amundsen.py:129]: sorry, what's the goal of this?

File: ingestion/src/metadata/ingestion/source/amundsen.py [hunk 2]
```diff
@@ -223,7 +294,7 @@ def create_dashboard_entity(self, dashboard):
             service_entity = get_dashboard_service_or_create(
```
Comment (pmbrull) [inline ingestion/src/metadata/ingestion/source/amundsen.py:294]: instead of get or create, we should just GET?

Comment (pmbrull) [reply ingestion/src/metadata/ingestion/source/amundsen.py:283]: if in the docs we specify that users should first create the services, then IMO we should:
1. try to get the service
2. if it is not there, throw an exception calling out that "service named XYZ of type ABC cannot be found"

Comment (ulixius9) [reply ingestion/src/metadata/ingestion/source/amundsen.py:283]: I'm logging an error in case of service not found, shall we throw error 🤔

Comment (pmbrull) [reply ingestion/src/metadata/ingestion/source/amundsen.py:283]: mmh logging + ignoring the entity might be good enough!

**Comment by pmbrull** [comment]
don't care about the sonar error here

**Pull request merged**